docs/monitoring/index.md

```diff
@@ -36,12 +36,13 @@ Follow along the [Quick Start](./quick-start) guide to learn how.
 
 App settings and actions include, but are not limited to:
 
-  - inviting others to your App
-  - [alert rules](../alerts)
   - changing your App's plan
-  - connecting Apps
-  - heartbeat alert cration
-  - App ownership transfer
+  - [managing alert rules](../alerts)
+  - [connecting Apps](https://sematext.com/docs/guide/integrations-guide/#connected-apps)
```

Comment:
@cpipilas Note there is a ticket in next sprint to add a page for Connected Apps.  We'll need to update any links to this section, like this one, when we do this.